map_server/src/main.cpp

```diff
@@ -91,25 +91,25 @@ class MapServer
 #endif
         try {
           doc["resolution"] >> res;
-        } catch (YAML::InvalidScalar) {
+        } catch (YAML::InvalidScalar &) {
```

Comment:
can you put them in a separate commit in this PR ? That'll be cleaner for history and rebasing